.prettierignore

```diff
@@ -0,0 +1,10 @@
+gondor/app/styles/_custom.scss
+gondor/app/styles/_mixins.scss
+gondor/app/styles/_nunito.scss
+gondor/app/styles/_variables.scss
+gondor/app/styles/_gogo.style.scss
+gondor/app/styles/theme/styles.scss
+gondor/app/styles/theme/simple-line-icons/css/simple-line-icons.css
+gondor/app/styles/theme/iconsmind/style.scss
+gondor/app/styles/theme
+gondor/app/styles/_variables.scss
```

Comment:
Why have you added `gondor` at the start of each line? also adding `app/styles/theme` should work for all the nested files.

Reply:
Sorry, i was checking something and then i forgot to remove this diff